downloads.html

```diff
@@ -131,7 +131,12 @@ <h2>macOS</h2>
     <p>macOS 10.10 or later is recommended.</p>
 
     <ul>
-      <li>Use <a href="http://brew.sh/">Homebrew</a> to get stable version <code>brew install minetest</code></li>
+      <li>Use <a href="http://brew.sh/">Homebrew</a> to get stable version: 
+        <ul style="list-style: none;">
+          <li><code>brew install minetest</code></li>
+          <li><code>brew linkapps</code></li>		
```

Comment:
"if i want to use ~/Application i have to use

brew linkapps --local"
https://github.com/Homebrew/legacy-homebrew/issues/8699#issuecomment-62160576

I am not sure if this is the right way of doing it. Will try on my mac tomorrow. On a PC right now.

Reply:
https://github.com/minetest/minetest.github.io/pull/119#issuecomment-371425277